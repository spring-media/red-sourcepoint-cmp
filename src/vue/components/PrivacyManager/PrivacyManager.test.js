import { mount } from '@vue/test-utils';
import { PrivacyManager } from './index';

describe('PrivacyManager component', () => {
  it('should provide a default scoped slot', () => {
    const wrapper = mount(PrivacyManager, {
      slots: {
        default: '<div>Privacy Manager default slot</div>',
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
