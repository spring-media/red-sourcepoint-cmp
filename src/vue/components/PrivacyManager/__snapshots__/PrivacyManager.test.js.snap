// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyManager component should provide a default scoped slot 1`] = `
<div>
  Privacy Manager default slot
</div>
`;
